### PR TITLE
Enable coop map previews and add leaderboard criteria

### LINKED ISSUE #2737: Add Leaderboard Criteria
On the Coop tab where the leaderboard Title is could we add a small I in a circle to provide a description to what qualifies to be on the leaderboard. As the leaderboard only show if the game is completed on hard with all secondary mission completed as well

image

### PR DESCRIPTION
**Pull request by Marc-Spector**
Closes #2737 

image

### CODE REVIEW

**Comment by Marc-Spector** [comment]
@MrRowey Is this criteria true?

**Comment by MrRowey** [comment]
> @MrRowey Is this criteria true?

It just need to be on hard difficulty secondary objective don't have to be met

**Comment by Marc-Spector** [comment]
I don't see the "no" value in "Secondary objectives" column in any missions

**Comment by Marc-Spector** [comment]
I found it. I did not search well =)

**Comment by MrRowey** [comment]
image
@Marc-Spector

File: src/main/java/com/faforever/client/chat/ChatUserService.java [hunk 1]
```diff
@@ -143,16 +143,11 @@ private void setGameImages(ChatChannelUser chatChannelUser, PlayerBean player) {
...
-      mapImage = mapService.loadPreview(player.getGame().getMapFolderName(), PreviewSize.SMALL);
-    } else {
-      mapImage = null;
-    }
     chatChannelUser.setStatusTooltipText(i18n.get(status.getI18nKey()));
     chatChannelUser.setGameStatusImage(playerStatusImage);
-    chatChannelUser.setMapImage(mapImage);
+    chatChannelUser.setMapImage(status != PlayerStatus.IDLE && player.getGame() != null
+        ? mapService.loadPreview(player.getGame().getMapFolderName(), PreviewSize.SMALL)
+        : null);
```
Comment (Sheikah45) [inline src/main/java/com/faforever/client/chat/ChatUserService.java]: This is a little long to do a ternary inline might be better for readability to still assign a local variable

**Review by Sheikah45 — CHANGES_REQUESTED** [review]

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**